src/olympia/migrations/1126-db-schema-fixes.sql

```diff
@@ -0,0 +1,186 @@
+    MODIFY `icontype` varchar(25) NOT NULL,
+    MODIFY `weeklydownloads` int(10) unsigned NOT NULL,
+    MODIFY `hotness` double NOT NULL,
+    MODIFY `experimental` tinyint(1) NOT NULL,
+    DROP FOREIGN KEY `addons_ibfk_2`,  /* name */
+    DROP FOREIGN KEY `addons_ibfk_3`,  /* homepage */
+    DROP FOREIGN KEY `addons_ibfk_4`,  /* description */
+    DROP FOREIGN KEY `addons_ibfk_5`,  /* summary */
+    DROP FOREIGN KEY `addons_ibfk_6`,  /* developercomments */
+    DROP FOREIGN KEY `addons_ibfk_7`,  /* eula */
+    DROP FOREIGN KEY `addons_ibfk_8`,  /* privacypolicy */
+    DROP FOREIGN KEY `addons_ibfk_9`,  /* supporturl */
+    DROP FOREIGN KEY `addons_ibfk_10`,  /* supportemail */
+    DROP FOREIGN KEY `addons_ibfk_14`,  /* current_version */
+    DROP KEY `addontype_id`,
+    DROP KEY `status`,
+    DROP KEY `inactive`,
+    DROP KEY `supporturl`,
+    DROP KEY `supportemail`,
+    DROP KEY `target_locale`,
+    DROP KEY `bayesianrating`,
+    DROP KEY `last_updated`,
+    DROP KEY `current_version`,
+    DROP KEY `created_idx`,
+    DROP KEY `modified_idx`,
+    DROP KEY `weeklydownloads_idx`,
+    DROP KEY `rating_type_idx`,
+    DROP KEY `created_type_idx`,
+    DROP KEY `modified_type_idx`,
+    DROP KEY `downloads_type_idx`,
+    DROP KEY `last_updated_type_idx`,
+    DROP KEY `type_status_inactive_idx`,
+    DROP KEY `hotness_idx`,
+    DROP KEY `visible_idx`,  /* (`addontype_id`,`status`,`inactive`,`current_version`) - note: no replacement */
+    DROP KEY `adus_type_idx`,
+    DROP KEY `name_2`;  /* (`name`,`status`,`addontype_id`) - note: no replacement */
+ALTER TABLE `addons`
+    ADD CONSTRAINT `addons_name_78bce5d2_fk_translations_id` FOREIGN KEY (`name`) REFERENCES `translations` (`id`),
+    ADD CONSTRAINT `addons_homepage_f34e15ae_fk_translations_id` FOREIGN KEY (`homepage`) REFERENCES `translations` (`id`),
+    ADD CONSTRAINT `addons_description_2300852e_fk_translations_id` FOREIGN KEY (`description`) REFERENCES `translations` (`id`),
+    ADD CONSTRAINT `addons_summary_0d397f7c_fk_translations_id` FOREIGN KEY (`summary`) REFERENCES `translations` (`id`),
+    ADD CONSTRAINT `addons_developercomments_b365508d_fk_translations_id` FOREIGN KEY (`developercomments`) REFERENCES `translations` (`id`),
+    ADD CONSTRAINT `addons_eula_62f9d8e4_fk_translations_id` FOREIGN KEY (`eula`) REFERENCES `translations` (`id`),
+    ADD CONSTRAINT `addons_privacypolicy_5e0b364a_fk_translations_id` FOREIGN KEY (`privacypolicy`) REFERENCES `translations` (`id`),
+    ADD CONSTRAINT `addons_supporturl_1101e07c_fk_translations_id` FOREIGN KEY (`supporturl`) REFERENCES `translations` (`id`),
+    ADD CONSTRAINT `addons_supportemail_da41fe48_fk_translations_id` FOREIGN KEY (`supportemail`) REFERENCES `translations` (`id`),
+    ADD CONSTRAINT `addons_current_version_5a2c8cb3_fk_versions_id` FOREIGN KEY (`current_version`) REFERENCES `versions` (`id`),
+    ADD KEY `addons_status_2e8bf8cf` (`status`),
```

Comment:
As mentioned on IRC on Friday: let's try to use [Django's index naming feature](https://docs.djangoproject.com/en/2.2/ref/models/indexes/#django.db.models.Index.name) to avoid re-creating those instead.